### PR TITLE
le-cinephile: add French private tracker. resolves #14553

### LINKED ISSUE #14553: [req]: Le Cinephile
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Private

### Tracker details

```markdown
- Name: Le Cinephile
- URL: https://le-cinephile.de/
- Description: Le Cinephile is a FRENCH Private Torrent Tracker for MOVIES / TV / GENERAL
- Minimum Seeding Time and Minimum Ratio (if Private): 60hrs
- Additional Comments: 
engine: 3xT
```

### PR DESCRIPTION
**Pull request by ilike2burnthing**
#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Issues Fixed or Closed by this PR

* Fixes #14553

### CODE REVIEW

File: src/Jackett.Common/Definitions/le-cinephile.yml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+  fields:
+    category_p1:
+      selector: div.kat_cat_pic_name
+    category_p2:
+      selector: div.kat_cat_pic_name_b
+    categorydesc:
+      text: "{{ .Result.category_p1 }} {{ .Result.category_p2 }}"
+    title:
+      selector: a.selection_a
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/le-cinephile.yml:152]:
```suggestion
    title_phase1:
      selector: a[href^="details.php?id="]
    title_multilang:
      text: "{{ .Result.title_phase1 }}"
      filters:
        - name: re_replace
          args: ["(?i)([\\s|\\.|-]multi[\\s|\\.|-])", ".{{ .Config.multilanguage }}."]
    title_phase2:
      text: "{{ if .Config.multilang }}{{ .Result.title_multilang }}{{ else }}{{ .Result.title_phase1 }}{{ end }}"
    title_vostfr:
      text: "{{ .Result.title_phase2 }}"
      filters:
        - name: re_replace
          args: ["(?i)([\\s|\\.|-]vostfr[\\s|\\.|-])", ".ENGLISH."]
        - name: re_replace
          args: ["(?i)([\\s|\\.|-]subfrench[\\s|\\.|-])", ".ENGLISH."]
    title:
      text: "{{ if .Config.vostfr }}{{ .Result.title_vostfr }}{{ else }}{{ .Result.title_phase2 }}{{ end }}"
```

**Review by garfield69 — COMMENTED** [review]
suggestions

File: src/Jackett.Common/Definitions/le-cinephile.yml [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      seeds: seeders
+      size: size
+      name: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
```
[the remark this answers is not in the document]
Comment (garfield69) [reply src/Jackett.Common/Definitions/le-cinephile.yml:98]: the faq address will need checking, I'm just winging it off the old fouducinema indexer

**Comment by garfield69** [comment]
oops, missed the boat ;-)

**Comment by ilike2burnthing** [comment]
Ah! Yes, forgot about all that. Will add now and post the today's build a lil late.

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/commit/dfc1cc5fb96d2dbc71706cb84e2db2d410666d3d

**Pull request merged**